GliaWidgetsTests/CallVisualizer/CallVisualizerTests.swift

```diff
@@ -0,0 +1,82 @@
+import XCTest
+import SalemoveSDK
+@testable import GliaWidgets
+
+final class CallVisualizerTests: XCTestCase {
+    func testScreenSharingStateChanged() {
+        enum Call { case updateState }
+        var calls: [Call] = []
```

Comment:
I'd like to propose to move enum definition and calls var on one level top.
```suggestion
final class CallVisualizerTests: XCTestCase {
    enum Call { case updateState }
    var calls: [Call] = []
    
    override func setUp() throws { 
        super.setUp()
        calls.removeAll()
     }
    
    func testScreenSharingStateChanged() {
```

Reply:
3337cd283a4de40419b8ccd27f3e4d4371aeb22d